cmake/cpack_settings.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+include(InstallRequiredSystemLibraries)
+set(CPACK_PACKAGE_NAME "ccf")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Confidential Consortium Framework")
+set(CPACK_PACKAGE_CONTACT "ccfeng@microsoft.com")
+set(CPACK_RESOURCE_FILE_LICENSE "${CCF_DIR}/LICENSE")
+set(CPACK_PACKAGE_VERSION ${CCF_RELEASE_VERSION})
+set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+
+# CPack variables for Debian packages
+set(CPACK_DEBIAN_PACKAGE_DEPENDS
+    "open-enclave (>=0.10.0)"
```

Comment:
I think we also want those, not sure which base versions yet:

  - libc++abi1
  - libc++1
  - libuv1
  - libcurl4